inc/3rd-party/plugins/autoptimize.php

```diff
@@ -107,6 +107,31 @@ function rocket_activate_autoptimize() {
 }
 add_action( 'activate_autoptimize/autoptimize.php', 'rocket_activate_autoptimize', 11 );
 
+/**
+ * Disable WP Rocket minification if Autoptimize css /js minification is enabled.
+ *
+ * @param array $options WP Rocket options array.
+ *
+ * @since 3.18
+ *
+ * @return array
+ */
+function rocket_maybe_disable_minification( $options ) {
+	if ( 'on' === get_option( 'autoptimize_css' ) ) {
+		update_rocket_option( 'minify_css', 0 );
```

Comment:
If we use the `rocket_first_install_options` filter, there shouldn't be any `update_rocket_option()` call since the option itself doesn't exist yet.

setting the array value to `0` should be enough.

Reply:
Totally agree with @remyperona 
@Khadreal can u replicate the issue mentioned by Mai? for me I can't (after removing the callback rocket_maybe_disable_minification)

![image](https://github.com/user-attachments/assets/b71bae4c-e10d-47a5-a650-4eb2c989070f)

Reply:
I pushed a small commit to fix this.